README.md

```diff
@@ -3,127 +3,139 @@
 ![Awesome](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg)
 
 ## Awesome CryptoNote
-A curated list of awesome CryptoNote open-source blockchains, resources, projects, and shiny other things related. 
+
+A curated list of awesome CryptoNote open-source blockchains, resources, projects, and shiny other things related.
 Many projects are derived from forks but none shall be listed if they're mere clones.
 
 > Pull Request's are **always** very welcome but this isn't a place to "shill".
 
-- [Awesome CryptoNote](https://github.com/lithe-project/awesome-cryptonote#awesome-cryptonote)
-  - [Application](https://github.com/lithe-project/awesome-cryptonote#application)
-  - [Blockchain](https://github.com/lithe-project/awesome-cryptonote#blockchain)
-  - [Documentation](https://github.com/lithe-project/awesome-cryptonote#documentation)
-  - [Graphical User Interface (GUI)](https://github.com/lithe-project/awesome-cryptonote#graphical-user-interfaces-gui)
-  - [Hashing Algorithm (PoW)](https://github.com/lithe-project/awesome-cryptonote#hashing-algorithm)
-  - [Mining Software](https://github.com/lithe-project/awesome-cryptonote#mining-software)
-  - [Pool Software](https://github.com/lithe-project/awesome-cryptonote#pool-software)
-  - [Web Wallet](https://github.com/lithe-project/awesome-cryptonote#web-wallet)
-  - [Misc.](https://github.com/lithe-project/awesome-cryptonote#misc)
-  
+-   [Awesome CryptoNote](https://github.com/lithe-project/awesome-cryptonote#awesome-cryptonote)
+
+-   [Application](https://github.com/lithe-project/awesome-cryptonote#application)
+-   [Blockchain](https://github.com/lithe-project/awesome-cryptonote#blockchain)
+-   [Documentation](https://github.com/lithe-project/awesome-cryptonote#documentation)
+-   [Graphical User Interface (GUI)](https://github.com/lithe-project/awesome-cryptonote#graphical-user-interfaces-gui)
+-   [Hashing Algorithm (PoW)](https://github.com/lithe-project/awesome-cryptonote#hashing-algorithm)
+-   [Mining Software](https://github.com/lithe-project/awesome-cryptonote#mining-software)
+-   [Pool Software](https://github.com/lithe-project/awesome-cryptonote#pool-software)
+-   [Web Wallet](https://github.com/lithe-project/awesome-cryptonote#web-wallet)
+-   [Misc.](https://github.com/lithe-project/awesome-cryptonote#misc)
```

Comment:
Can you revert this bit, please?

Currently, the layout of the document is set out like;
- title
    - subtitle
    - subtitle

but the new solution makes them all titles